metals/src/main/scala/scala/meta/internal/metals/doctor/DoctorResults.scala

```diff
@@ -152,3 +153,19 @@ final case class DoctorFolderHeader(
     base
   }
 }
+
+/**
+ * Information about an error report.
+ * @param name display name of the error
+ * @param timestamp date and time timestamp of the report
+ * @param uri error report file uri
+ * @param buildTarget optional build target that error is associated with
+ * @param shortSummary short error summary
```

Comment:
Is the short summary the stack trace? It might be nice to actually differentiate any informational message from code/stacktrackes. That way when we display the stack trace we can actually display it as a code snippet. In theory if we wanted to get crazy we could even use the stack trace analyzer on it and add links right from the doctor stacktrace into the code. Either way, the reason I mention this is right now it seems like you might have a short message and a stacktrace in here, when it'd be nice to differentiate those to make sure the stacktrace is displayed as code if the client can do that.

Reply:
It's not the stack trace, just the error message. Short summary is supposed to be short.

Reply:
ahh ok, then I misunderstood.